prisma/schema.prisma

```diff
@@ -278,6 +279,18 @@ model WalletCreator {
   @@id([creatorSlug, walletAddress])
 }
 
+model Listing {
+  nft   Nft @relation(fields: [mint],references: [address])
+  mint  String @id
+  tradeStateAddress String
+  sellerAddress String
+  price Int
+  feePayer String
+  signature String
+  createdAt DateTime
+  canceledAt DateTime?
```

Comment:
I wonder if we should use `createdAt` or `listedAt` here? Don't think we necessarily should, just a random thought...
Maybe best to leave it as is

Reply:
yeah! i named these as such because of required type we needed in finding listing